### DISCUSSION TITLE
Add more information about a complete printer calibration

### DISCUSSION THREAD

**Discussion by cpasoft** [Guilouz/Klipper-Flsun-Speeder-Pad#64] [Ideas]
It would be helpful to have a step-by-step guide for a thorough and accurate calibration process, including more advanced options that are currently not available, such as "enhanced delta calibration." Additionally, providing more detailed explanations for procedures that are only briefly mentioned, like "EndStop Calibration," would greatly benefit users.

The calibrate your printer section already mentions this procedures:
*    Start BED PID and save configuration.
*    Start HOTEND PID and save configuration.
Both not explained at all

*    Calibrate your extruder by calculing Rotation Distance.
This is already explained. I'll include the link.

*    Start ENDSTOP CALIBRATION and save configuration.
Not explained at all

*    Start DELTA CALIBRATION and save configuration.
*    Start BED LEVELING and save configuration.
*    Adjust Z-OFFSET, first you need to move to Z=0 and then adjust nozzle position with a sheet of paper.
Even though they are not explained here, are usually covered in the printer manual as basic calibration procedures that everyone should be aware of. However, it wouldn't hurt to provide some additional explanation for those who may not be familiar with them.

I think it would useful add information about:
+ Enhanced delta calibration
+ Pressure advance (already present on the useful links)
+ Retractions? Not sure if this would fit under this section...
+ Flow rate calibration
+ others??

Thanks for all the work that has been put into this repository... My congratulations for that!!...

Regards!

**Comment by Guilouz** [comment]
All of them are explain in Klipper documentation.

**Reply by cpasoft** [reply]
I know that, but it would be nice to have a well-organised list with links to their respective documentation. 

For example, when it says "Endstop calibration", what is it referring to exactly??

Anyway, it's just a suggestion. I don't want you take it as criticism or anything like that. I think the work done here is excellent.

**Comment by cpasoft** [comment]
Just want to say Thank you for the huge improvement of this section...

THANK YOU VERY VERY MUCH!!!

**Comment by Gazzavr** [comment]
The whole procedure to obtain an up-to-date SR was outstanding, Guilouz, also for less linux-savvy people like me : being taken care step by step was really encouraging...but when I met the Calibration section it was like  the style was totally different and less helpful than it was before : I even smashed the nozzle on the plate at a certain point, because the first part of calibration help ( Z_OFFSET ) takes care of suggesting when to apply the leveling sensor, but then ( DELTA / BED LEVELING ) it does not.
And I'm certain a lot of people could laugh right now because to them the fact is known, but this was the first time for me to calibrate this way...all the other environments ( Marlin or original SP ) have much clear and safe calibration procedures ( although probably less in-depth ).

**Reply by 808mp5** [reply]
I thocked into the bed as well... they mention in the calibration when to take the probe off.... but don't tell you to put it back on when you continue with the guide... ruined a brand new PEI plate that never even tasted filament due to lack of experienced/guidance
